test/Interactions/test_interactions.py

```diff
@@ -1074,3 +1075,19 @@ def test_morse():
     range = 7.56
     pot = Morse(w, range)
     assert pot.evaluate(0)[0] == - w
+    
+def test_electrostatic_coulomb():
```

Comment:
The test will not fail if the output of the function is wrong. 

1. add an assertion that the values are correct (e.g. extract some of the values from their plot and compare to them, doesn't need to be super accurate). 

2. wrap the plotting into an `if False:` so that the plot will not show up in normal testing sessions (would just be here for debugging purposes)

Reply:
Ah sorry just read the comment in the PR. if there is no such data then leave it. But wouldn't fig. 3 do part of the job ?

Reply:
1. Took test values and plotted, they look exactly as Fig. 3 now after non-dimensionalization
2. Plot wrapped in if False